src/pal/src/exception/seh-unwind.cpp

```diff
@@ -108,7 +108,7 @@ static void WinContextToUnwindContext(CONTEXT *winContext, unw_context_t *unwCon
     unwContext->regs[12] = 0;
     unwContext->regs[13] = winContext->Sp;
     unwContext->regs[14] = winContext->Lr;
-    unwContext->regs[15] = winContext->Pc;
+    unwContext->regs[15] = winContext->Pc & ~0x1;
```

Comment:
I wonder why this is needed - I know it was getting the Pc with bit 0 cleared before this change due to the fact that it was cleared in UnwindContextToWinContext. I just wonder how the libunwind handles code with mixed ARM and THUMB frames in general.